MediaBrowser.Providers/Plugins/TheTvdb/TvdbClientManager.cs

```diff
@@ -229,6 +229,51 @@ await GetEpisodesPageAsync(Convert.ToInt32(seriesTvdbId), episodeQuery, language
             return GetEpisodesPageAsync(tvdbId, 1, episodeQuery, language, cancellationToken);
         }
 
+        public async Task<IEnumerable<KeyType>> GetImageKeyTypesForSeriesAsync(int tvdbId, string language, CancellationToken cancellationToken)
+        {
+            var cacheKey = GenerateKey(nameof(TvDbClient.Series.GetImagesSummaryAsync), tvdbId);
+            var imagesSummary = await TryGetValue(cacheKey, language, () => TvDbClient.Series.GetImagesSummaryAsync(tvdbId, cancellationToken)).ConfigureAwait(false);
+            var keyTypes = new List<KeyType>();
+
+            if (imagesSummary.Data.Fanart > 0)
+            {
+                keyTypes.Add(KeyType.Fanart);
```

Comment:
```suggestion
                yield return KeyType.Fanart;
```

Reply:
Just fyi, this won't work with a Task return type. Doesn't matter though. I updated this way before your review comments, but github refuses to update the PR.